conf/generators/assignment.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "assignment_op",
+	"fname": "11.4.1--{1}_assignment.sv",
+	"templ": [
+		"/*",
+		":name: {1}_assignment",
+		":description: {0} assignment test",
+		":should_fail: 0",
+		":tags: 11.4.1",
+		"*/",
+		"module top();",
+		"int a = 12;",
+		"int b = 5;",
+		"initial begin",
+		"    a {0} b;",
+		"end",
+		"endmodule"
+	],
```

Comment:
Why does this have to be an array?

Reply:
Is it possible to use multiline variables in JSON?
When I tried to use a single string taking multiple lines it failed parsing in the generator script